fileutil/file.go

```diff
@@ -95,3 +96,17 @@ func DownloadToMemory(url string) ([]byte, error) {
 	}
 	return contents, nil
 }
+
+// RemoveFileMatches for the given filepath pattern.
+func RemoveFileMatches(pattern string) error {
+	files, err := filepath.Glob(pattern)
+	if err != nil {
+		return err
+	}
+	for _, f := range files {
+		if err := os.Remove(f); err != nil {
+			return err
```

Comment:
This is an anti-pattern. Operations on sequences should always be either of two things:
     - atomic
     - as complete as possible
We should change this to return some sort of progress (either succeeded and failed files) or log the errors. Otherwise the operation has been left in an indeterminate partially completed state where there may or may not be additional files the can be removed.

Reply:
Oh duh. Pushed a commit with the logging solution